templates/includes/markdown/get_started_first_boot.md

```diff
@@ -18,15 +18,13 @@ the same network, using the following command:
 
 The user name is your Ubuntu SSO user name, it has been reminded to you at the end of the account configuration step.
 
-##### First boot tips
+#### First boot tips
 
   * During setup,` console-conf` will download the SSH key registered with your Store account and configure it so you can log into the device via `ssh <Ubuntu SSO account name>@<device IP address>` without a password.
   * There is no default `ubuntu` user on these images, but you can run `sudo passwd <account name>` to set a password in case you need a local console login.
 
-### Install and develop snaps
+## Install and develop snaps
 
-Your board is ready to have snaps installed.
-
-[Get started with the snap command](http://snapcraft.io/docs/core/usage)
+Your board is ready to have snaps installed, [get started with the `snap` command](http://snapcraft.io/docs/core/usage).
```

Comment:
Should this have a color or an mdash? "...have snaps installed — get started..."? I feel the comma isn't quite the right punctuation.